### ISSUE TITLE
Question on the experimental setting of frozen model.

### ISSUE THREAD

**Issue by SCZwangxiao** [OpenGVLab/UniFormerV2#34]
The CLIP backbone is not frozen by setting `UNIFORMERV2.FROZEN = False`. The gradient will still be backward into the CLIP model, since the `temporal_cls_token` acts as a query to attend all CLIP image patches.

The CLIP backbone parameters have also changed at the checkpoints you provided, such as [Frozen UniFormerV2-L/14@336](https://pjlab-gvm-data.oss-cn-shanghai.aliyuncs.com/uniformerv2/k710/frozen_k710_uniformerv2_l14_8x336.pyth).

**Comment by Andy1621** [comment]
Thanks for your finding! I have ignored adding the code about `frozen` when merging code. I will add it soon.

1. Note that `frozen` means prompt tuning, thus the gradient of new parameters will be backward.
2. I have checked the backbone parameters, and they are the same as the original visual encoder. Maybe you use different packages or environments (FP16?), thus the extracted weights of the visual encoder are a little different.

The frozen UniFormerV2:
<img width="456" alt="image" src="https://user-images.githubusercontent.com/24236723/223452724-946b4c87-2d9b-4b03-8951-a6fadf174c93.png">

The original CLIP visual encoder:
<img width="586" alt="image" src="https://user-images.githubusercontent.com/24236723/223452840-d90f97b3-9a68-422f-b9ce-c15152045f6c.png">
